src/cfp.md

```diff
@@ -0,0 +1,284 @@
+---
+#title: Call For Proposal
+#description: Call For Proposal
+layout: page.njk
+---
+
+<style>
```

Comment:
Instead of page wise styling, can you write a common stylesheet file to maintain consistency ?